.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,154 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  
+  cmake_test:
+    
+    runs-on: ubuntu-latest
```

Comment:
I would leave out the newlines within each job. They break up the configuration in a way that makes it hard to know what the configuration of a job is at a glance, which is the most important

Reply:
Fixed. I agree it does make it a bit harder to understand at first glance. Thanks!